trino/client.py

```diff
@@ -456,7 +456,7 @@ def statement_url(self) -> str:
     def next_uri(self) -> Optional[str]:
         return self._next_uri
 
-    def post(self, sql, additional_http_headers=None):
+    def post(self, sql: str, additional_http_headers: Optional[Dict[str, Any]] = None):
```

Comment:
As of yet I'm unsure of the return type. This should become clearer when the module has type enforcement. This method (as well as `get`) are required to have their arguments typed given they're called in `trino/transaction.py` and we have `disallow_untyped_calls = true` set.